src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/AlleleSubsettingUtils.java

```diff
@@ -55,7 +55,7 @@ public static GenotypesContext subsetAlleles(final GenotypesContext originalGs,
                                                  final GenotypePriorCalculator gpc,
                                                  final GenotypeAssignmentMethod assignmentMethod,
                                                  final int depth,
-                                                 final boolean suppressUninformativePLs) {
+                                                 final boolean forceEmitUninformativePLs) {
```

Comment:
I took this out instead.  The reason it was still there was because I didn't want to change the SelectVariants integration test expected outputs, but that's not a good reason.  I want to prefer explicitness and consistency over space savings or whatever the motivation was for dropping PLs sometimes.